Sources/Text/Text.swift

```diff
@@ -30,7 +30,6 @@ extension Warp {
         public var body: some View {
             SwiftUI.Text(text)
                 .font(from: style.asTypography)
-                .foregroundColor(colorProvider.labelText)
```

Comment:
if you remove this then other places that uses our own Warp.Text suddenly lost their colors and became .black, right?

Reply:
Good point, I will revert this.